File: 1.0-computer_science/1.0-cs.c/0.0-data-structure/callenges/2.0-search.js
function searchArray(array, elem){
    var size = 0;
    while (array[size] != undefined) size += 1;
    for (var i = 0; i < size; i++)
        if (array[i] === elem) return `The element is in the array in the position: ${i}`;
    return 'The element is not in the array';
}

var array = [1, 2, 3, 4, 5];
var elem = 4;
console.log(searchArray(array, elem));

File: 1.0-computer_science/1.0-cs.c/0.0-data-structure/callenges/2.0-search.py
def searchArray(array, elemt):
    size = len(array)
    for i in range (size):
        if (array[i] == elemt):
            return f'The element is in array in the position: {i}'
    return 'The element is not in array'


if __name__ == "__main__":
    array = [1, 2, 3, 4, 5]
    element = 4
    print(searchArray(array, element))